### PR TITLE
домашня робота 09, поки що без додаткових завдань.

### PR DESCRIPTION
**Pull request by LilianaLukash**
домашня робота 09, поки що без додаткових завдань.

### CODE REVIEW

File: home-work9.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+def f_generator(number):
+    numb1, numb2 = 0, 1
+    for _ in range(number):
+        yield numb1
+        numb1, numb2 = numb2, numb1+numb2
+
+inp = int(input("Enter integer for Fibonacci Generator "))
+gen1 = f_generator(inp)
+for _ in range(inp):
+    print(next(gen1))
```
Comment (klymenok) [inline home-work9.py]: The task was to print only one number of sequence but not all of them

Comment (klymenok) [reply home-work9.py]: Also, I would suggest to start counting from zero:  0 = 0, 1 = 1, 2 = 1, 3 = 2, 4 = 3, 5 = 5, 6 = 8, 7 = 13

**Review by klymenok — CHANGES_REQUESTED** [review]

File: home-work9.py (hunk 1 above)
Comment (LilianaLukash) [reply home-work9.py]: https://github.com/LilianaLukash/PythonStudy/pull/6/commits/663793f1bce511b03ebded6cd15d8616ce34a790
Please see file with printing only one number.

Program already was printing from 0

File: regression9.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    if rnumb == 1:
+        return 0
+    elif rnumb ==2:
+        return 1
+    else:
+        nfib = FibRecursion(rnumb-1)+FibRecursion(rnumb-2)
+        return nfib
+
+inpR = int(input("Enter integer for Fibonacci Recursion"))
+for i in range(inpR):
```
Comment (klymenok) [inline regression9.py]: I didn't get why do you use `for` here? You can just print `print(FibRecursion(inpR))` to get a value that you need

File: regression9.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    next= FibRecursion(i+1)
+    list.append(next)
+flist = FibRecursion(inpR)
```
Comment (klymenok) [inline regression9.py]: The result of your function is an integer, but you are trying to get an element from it, like from a list. This breaks your program

**Review by klymenok — CHANGES_REQUESTED** [review]

File: regression9.py (hunk 3 above)
Comment (LilianaLukash) [reply regression9.py]: Sorry, was experimenting and forgot to commit last version of this file. Now everything is fine

File: regression9.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# create Fibonacci Recursion
+list=[]
+def FibRecursion(rnumb):
+    if rnumb == 1:
+        return 0
+    elif rnumb ==2:
+        return 1
+    else:
+        nfib = FibRecursion(rnumb-1)+FibRecursion(rnumb-2)
```
Comment (klymenok) [inline regression9.py:9]: You can avoid creating a new variable here by using return statement right here:
`return FibRecursion(rnumb-1)+FibRecursion(rnumb-2)`

**Review by klymenok — APPROVED** [review]
Looks good, check my comment

**Pull request merged**